packages/vaadin-checkbox/src/vaadin-checkbox-group.js

```diff
@@ -90,7 +90,7 @@ class CheckboxGroupElement extends ThemableMixin(DirMixin(PolymerElement)) {
 
         <div
           part="helper-text"
-          on-click="focus"
+          on-click="_handleClick"
```

Comment:
Shouldn't this be removed as a whole? See e.g. @rolfsmeds comment: https://github.com/vaadin/flow-components/issues/955#issuecomment-757970974

This would reduce the logic and possible side-effects if somebody clicks on the helper-text to select/copy it and the user is wondering why he can't select it.

Reply:
Yes, that was an option too, but we can figure out where click was on `slot` with name of helper or just a text, this is not fixed yet and is open for discussion.

Reply:
Actually, it was my suggestion to implement it this way.
However, now when I'm thinking about it, we probably should indeed remove these listeners for simplicity.

Reply:
Fight for simplicity :vaadin: